### ISSUE TITLE
Modify the SMCalloutView

### ISSUE THREAD

**Issue by hasanadil** [mapbox/DEPRECATED-mapbox-ios-sdk#462]
It would be good to have a way to set properties on the SMCalloutView from outside of the SDK. Right now, to get the annotation callout the way I'd like to have it, I am modifying the source in RMMapView.selectAnnotation: where the callout view is created. 

Would be nice to set properties on the RMMapView and it act as a proxy for SMCallout stuff.

*The default (via the callout's core graphics implementation) is what map box has set, and callout view draws more like iOS6 than 7 style.

**Comment by incanus** [comment]
> Would be nice to set properties on the RMMapView and it act as a proxy for SMCallout stuff.

Yes, though we will probably wait for post-2.0/GL to tackle this, which is basically just exposing existing SMCallloutView API. 

> *The default (via the callout's core graphics implementation) is what map box has set, and callout view draws more like iOS6 than 7 style.

Not sure I follow -- this should have a dual 6/7 appearance based on runtime environment.

**Comment by incanus** [comment]
![screen shot 2014-07-01 at 11 09 55 am](https://cloud.githubusercontent.com/assets/17722/3447220/f7c6b5aa-014a-11e4-9f71-03ff90841744.png)

![screen shot 2014-07-01 at 11 10 11 am](https://cloud.githubusercontent.com/assets/17722/3447222/fa5f70b8-014a-11e4-935d-e493af88a7ef.png)

**Comment by hasanadil** [comment]
I'll debug it and also see whats going on, am getting the old style, black gloss on 7.0-7.1 builds.
![screen shot 2014-07-01 at 9 56 32 pm](https://cloud.githubusercontent.com/assets/4443086/3451082/32f0b70c-018c-11e4-9514-9b1bdf6d8d85.png)

**Comment by hasanadil** [comment]
I had an older version of the calloutView since it installed from the default pods, its fixed in the latest version of it.
